src/app-bundles/data-entry-bundle.jsx

```diff
@@ -193,7 +193,7 @@ export default {
       const url = `/psapi/moriverDataEntry${queryFromObject(params)}`;
 
       apiGet(url, (err, body) => {
-        if (!err) {
+        if (!err && body.status) {
```

Comment:
Add the question marks in all changed files

```suggestion
        if (!err && body?.status) {
```